Extensions/ArtifactEngine/E2ETests/E2E.ts

```diff
@@ -266,4 +266,52 @@ describe('e2e tests', () => {
                 done();
             });
     })
+
+    var runWindowsBasedTest = process.platform == 'win32' ? it : it.skip;
```

Comment:
add this test to fileshare.e2e.tests. Add corresponding tests for vsts.e2e.tests and jenkins.e2e.tests.